src/net-questdb-client-tests/HttpTests.cs

```diff
@@ -40,6 +40,144 @@ public class HttpTests
     private const int HttpPort = 29473;
     private const int HttpsPort = 29474;
 
+    [Test]
+    public async Task BasicArrayDouble()
+    {
+        using var server = new DummyHttpServer(withBasicAuth: false);
+        await server.StartAsync(HttpPort);
+        var sender =
+            Sender.New(
+                $"http::addr={Host}:{HttpPort};username=asdasdada;password=asdadad;tls_verify=unsafe_off;auto_flush=off;");
+        await sender.Table("metrics")
+                    .Symbol("tag", "value")
+                    .Column("number", 10)
+                    .Column("string", "abc")
+                    .Column("array", new[] { 1.2, 2.6, 3.1, })
+                    .AtAsync(new DateTime(1970, 01, 01, 0, 0, 1));
+
+        await sender.SendAsync();
+        Assert.That(
+            server.PrintBuffer(),
+            Is.EqualTo("metrics,tag=value number=10i,string=\"abc\",array==ARRAY<3>[1.2,2.6,3.1] 1000000000\n"));
+        await server.StopAsync();
+    }
+
+    [Test]
+    public async Task BasicArrayDoubleNegotiationFailed()
+    {
+        using var server = new DummyHttpServer(withBasicAuth: false);
+        await server.StartAsync(HttpPort, new[] { 1, });
```

Comment:
If I comment out this line, the test still passes. I would expect it to fail with something like  "cannot connect" instead.